package.json

```diff
@@ -7,8 +7,8 @@
     "test": "jest --coverage",
     "start-dev": "concurrently \"npm run dev\" \"npm run css-dev\"",
     "start-prod": "concurrently \"npm run prod\" \"npm run css-prod\"",
-    "css-dev": "TAILWIND_MODE=watch postcss css/styles.css -o src/styles.css -w --verbose",
-    "css-prod": "NODE_ENV=production postcss css/styles.css -o src/styles.css",
+    "css-dev": "TAILWIND_MODE=watch postcss src/styles.css -o dist/styles.css -w --verbose",
+    "css-prod": "NODE_ENV=production postcss src/styles.css -o dist/styles.css",
     "dev": "npx webpack -w --config webpack.config.js --mode development",
     "prod": "npx webpack -w --config webpack.config.js --env production",
     "deploy": "gh-pages -d dist",
```

Comment:
- [ ] The scripts are not correctly set up,  since you are using webpack the reviewer should be able to open the project in localhost.